js/data/symbol_bucket.js

```diff
@@ -404,48 +488,19 @@ SymbolBucket.prototype.addToDebugBuffers = function(collisionTile) {
                 var maxZoom = Math.max(0, Math.min(25, this.zoom + Math.log(box.maxScale) / Math.LN2));
                 var placementZoom = Math.max(0, Math.min(25, this.zoom + Math.log(box.placementScale) / Math.LN2));
 
-                addCollisionVertex(buffer, anchorPoint, tl, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, tr, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, tr, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, br, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, br, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, bl, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, bl, maxZoom, placementZoom);
-                addCollisionVertex(buffer, anchorPoint, tl, maxZoom, placementZoom);
-
-                this.elementGroups.collisionBox.current.vertexLength += 8;
+                this.add.collisionBox.vertex(anchorPoint, tl, maxZoom, placementZoom);
```

Comment:
Y, I think that will have a positive impact on perf. Still trying to decide between that approach and writing these functions to top-level class methods (i.e. `addCollisionBoxVertex`).